sandbox/seccomp/compiler.go

```diff
@@ -31,14 +31,19 @@ import (
 )
 
 var (
-	// version-info format: <build-id> <libseccomp-version> <hash> <features>
-	// Where, the hash is calculated over all syscall names supported by the
-	// libseccomp library. The build-id is a 160-bit SHA-1 (40 char) string
-	// and the hash is a 256-bit SHA-256 (64 char) string. Allow libseccomp
-	// version to be 1-5 chars per field (eg, 1.2.3 or 12345.23456.34567)
-	// and 1-30 chars of colon-separated features.
-	// Ex: 7ac348ac9c934269214b00d1692dfa50d5d4a157 2.3.3 03e996919907bc7163bc83b95bca0ecab31300f20dfa365ea14047c698340e7c bpf-actlog
-	validVersionInfo = regexp.MustCompile(`^[0-9a-f]{1,40} [0-9]{1,5}\.[0-9]{1,5}\.[0-9]{1,5} [0-9a-f]{1,64} [-a-z0-9:]{1,30}$`)
+	// version-info format: <build-id> <libseccomp-version> <hash>
+	// <features> Where, the hash is calculated over all syscall names
+	// supported by the libseccomp library. The build-id is a string of up
+	// to 166 chars, accommodates 128-bit MD5 (16 chars), 160-bit SHA-1 (20
```

Comment:
Before you said the build-id was a 160-bit SHA-1 was '(40 char)', but now you say '20' but the testsuite still uses 40 char strings. Furthermore the '128-bit MD5' is using 32 chars in the testsuite. Can you update this comment?

Reply:
Good catch. I'll force push if you don't mind